modules/nodes-pods-autoscaling-custom-creating-trigger-auth.adoc

```diff
@@ -70,9 +70,9 @@ metadata:
 spec:
   scaleTargetRef:
     name: example-deployment
-  maxReplicaCount: 100
-  minReplicaCount: 0
-  pollingInterval: 30
+  maxReplicaCount: '100'
+  minReplicaCount: '0'
+  pollingInterval: '30'
   triggers:
```

Comment:
please revert this, the quotoes are only needed for values in `spec.triggers.metadata`